uniffi_bindgen/src/bindings/kotlin/gen_kotlin/mod.rs

```diff
@@ -903,6 +905,78 @@ mod filters {
         let spaces = usize::try_from(*spaces).unwrap_or_default();
         Ok(textwrap::indent(&wrapped, &" ".repeat(spaces)))
     }
+
+    fn serializable_type(ty: &Type, ci: &ComponentInterface) -> Result<bool, askama::Error> {
```

Comment:
Would it be okay to make `ComponentInterface::iter_types_in_item` public and use it here?

Reply:
yeah, that would be fine and better than copying it